### PR TITLE
singlecrystal: 5.0.0 (new cask)

### PR DESCRIPTION
**Pull request by singularitti**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [ ] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=pullrequests).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

**Comment by singularitti** [comment]
1. I was unable to find a link like `https://crystalmaker.com/support/updates/VersionData-SGM.xml`, please help me with the livecheck.
2. The `brew style --fix` will throw an error because `desc` contains "Single-crystal", but it is a scientific term, and I don't think that duplication matters.

**Review by samford — COMMENTED** [review]
> 1. I was unable to find a link like `https://crystalmaker.com/support/updates/VersionData-SGM.xml`, please help me with the livecheck.

Looks like the URL is https://crystalmaker.com/support/updates/VersionData-SCM.xml. I pushed a commit to add a `livecheck` block, using the approach in https://github.com/Homebrew/homebrew-cask/pull/172222/files#r1581568335.

Past the `livecheck` block, I'll leave the overall cask review to our cask maintainers.

**Comment by singularitti** [comment]
Ah, it was my oversight. Thank you so much @samford!

File: Casks/s/singlecrystal.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
+cask "singlecrystal" do
+  version "5.0.0"
+  sha256 :no_check
+
+  url "https://crystalmaker.com/downloads/singlecrystal#{version.major}_mac.zip"
+  name "SingleCrystal"
+  desc "Single-crystal diffraction software"
```
Comment (razvanazamfirei) [inline Casks/s/singlecrystal.rb:7]:
```suggestion
  desc "Crystal diffraction software"
```

**Review by razvanazamfirei — APPROVED** [review]

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**